### PR TITLE
Create Reaper Manager

### PR DESCRIPTION
**Pull request by brianmodel**
Created the reaper manager, which is a controller for reapers.

### CODE REVIEW

File: pkg/integration_tests/test_utils.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+// ReadConfigFile reads the config.json file, and returns the ProjectID and accessToken
+// specified in the file.
+func ReadConfigFile() (string, string) {
+	var configData TestConfig
+	jsonConfigFile, err := os.Open("config.json")
+	defer jsonConfigFile.Close()
+	if err != nil {
+		log.Fatal(err)
```
Comment (supercclank) [inline pkg/integration_tests/test_utils.go:37]: Prefer returning on error rather than exiting like this

File: pkg/reaper/reaper.go [hunk 2]
```diff
@@ -215,3 +252,15 @@ func maxTTL(resourceA, resourceB *resources.WatchedResource) (string, error) {
 		return resourceB.TTL, nil
 	}
 }
+
+// parseSchedule parses the cron time string that defined the reaper's
+// run schedule, and either returns a Schedule struct, or nil if the
+// schedule string is malformed.
+func parseSchedule(schedule string) cron.Schedule {
```
Comment (supercclank) [inline pkg/reaper/reaper.go:259]: Do we want to return an error here in addition to nil in the error found case?

File: pkg/integration_tests/test_utils.go (hunk 1 above)
Comment (brianmodel) [reply pkg/integration_tests/test_utils.go]: Fixed

File: pkg/reaper/reaper.go (hunk 2 above)
Comment (brianmodel) [reply pkg/reaper/reaper.go]: Function now returns schedule and error

**Pull request merged**